__tests__/__integration__/DatasetTree.integration.test.ts

```diff
@@ -236,15 +236,15 @@ describe("DatasetTree Integration Tests", async () => {
     /*************************************************************************************************************
      * Recently-opened member function tests
      *************************************************************************************************************/
-    it("Tests that addRecall adds a recently-opened file to the list", async () => {
-        testTree.addRecall(`[${sessNode.getLabel()}]: ${pattern}.EXT.SAMPLE.PDS(TESTMEMB)`);
-        const recall = testTree.getRecall();
+    it("Tests that addFileHistory adds a recently-opened file to the list", async () => {
+        testTree.addFileHistory(`[${sessNode.getLabel()}]: ${pattern}.EXT.SAMPLE.PDS(TESTMEMB)`);
+        const recall = testTree.getFileHistory();
```

Comment:
How about renaming `recall` to `lastDataset` ?

Reply:
Eeeee but right now all the `FileHistory` functions have matching names...and it would be nice if it had the format of `sometypeofHistory()` because then it will be matching with the other history function...addSearchHistory, getSearchHistory, etc.

Why do you want to rename it? Maybe we can find a better name that still matches with this format?

Reply:
The reason why I suggested to rename is that identifiers should describe the meaning of the variable, function ... and `recall` does give me any idea what it is about.

Reply:
Maybe there is a misunderstanding, here, I was referring to the variable name, not to the function name.

Reply:
I think I just failed to remove all the old names from the test files